packages/web-twig/src/Resources/components/Tooltip/__tests__/__snapshots__/tooltipPopover.twig.snap.html

```diff
@@ -5,12 +5,12 @@
     </title>
   </head>
   <body>
-    <div class="Tooltip is-hidden" data-spirit-placement="bottom">
+    <div class="Tooltip is-hidden" data-spirit-placement="bottom" data-spirit-trigger="click, hover">
       Hello there!
     </div>
     <!-- Render with all props -->
 
-    <div class="Tooltip Tooltip--dismissible is-hidden" id="my-tooltip" data-spirit-placement="right-start" data-spirit-placement-controlled="" data-spirit-enable-flipping="true" data-spirit-enable-flipping-cross-axis="true" data-spirit-flip-fallback-axis-side-direction="top, left, right, bottom" data-spirit-flip-fallback-placements="left-start" data-spirit-enable-shifting="true" data-spirit-enable-sizing="true" data-spirit-trigger="click, hover">
+    <div class="Tooltip Tooltip--dismissible is-hidden" id="my-tooltip" data-spirit-placement="right-start" data-spirit-flip-fallback-axis-side-direction="top, left, right, bottom" data-spirit-flip-fallback-placements="left-start" data-spirit-trigger="click, hover">
```

Comment:
wait what. The `enableFlipping` which is actually turned on by default should output `data-spirit-enable-flipping="true"` here. And it doesn't. Something is wrong.

Reply:
It was because it is by default set to true and doesn't need to be there for the JS plugin.
Changed to be there for both options now. Could be seen in new snapshot.